### DISCUSSION TITLE
[HOT TOPIC] 🔥: Addition of Tailwind CSS

### DISCUSSION THREAD

**Discussion by tamalCodes** [karmacircleCommunity/KarmaCircle#1153] [Ideas]
Currently, we are using just CSS with very small features of `Twitter Bootstrap` v5.3 there is a lot of repetitive CSS in the codebase but would it be a good idea to use Tailwind CSS in this case ? I really want to know what you all think.

We need to keep a few things in mind.

 - The styling must be done the same way as in any industry-grade work.
 - The code must be not too much cluttered with all the Tailwind classes.
 - There must be very less use of repetitive CSS throughout the codebase.

Keeping these in mind would it be a good idea to add the Tailwind ?

**Comment by Lokendrakushwah12** [comment]
yes i think tailwind would work best but problem may be that till now raw CSS was used so we can either use tailwind from now or also convert previous CSS in tailwind but it would take more time

**Reply by tamalCodes** [reply]
Great thought, instead of having a mix i think just having Tailwind would be a good idea ?

**Reply by Lokendrakushwah12** [reply]
yep i would prefer only having Tailwind

**Comment by Piyush-linux** [comment]
Should we setup an tailwind branch to work with , am familiar with tailwind . @tamalCodes
